### PR TITLE
- Update recording tags to be more consistent with Kodi (Matrix)

### PR DESCRIPTION
**Pull request by rbarthen**
- add episode number
- add series number
- add Episode name
- remove episode name from primary series title
- remove episode name from plot outline

### CODE REVIEW

File: src/pvrclient-argustv.cpp [hunk 1]
```diff
@@ -720,20 +720,12 @@ PVR_ERROR cPVRClientArgusTV::GetRecordings(bool deleted,
...
-                recording.Transform(true);
-                tag.SetDirectory(
-                    recordinggroup
-                        .ProgramTitle()); //used in Kodi as directory structure below "Server X - hostname"
-              }
+                tag.SetDirectory(recording.Title()); 
               else
-              {
-                recording.Transform(false);
                 tag.SetDirectory("");
```
Comment (ksooo) [inline src/pvrclient-argustv.cpp]: This line can be removed as empty directory is the default.

Comment (phunkyfish) [reply src/pvrclient-argustv.cpp]: That line is not part of the PR though 😉

But I get your point. There is no else required at all now.

Comment (ksooo) [reply src/pvrclient-argustv.cpp]: Exactly.

Comment (rbarthen) [reply src/pvrclient-argustv.cpp]: I removed the unnecessary else statement that set directory to empty string.

**Review by phunkyfish — APPROVED** [review]
Nice

File: pvr.argustv/addon.xml.in [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.argustv"
-  version="19.1.1"
+  version="19.1.2"
```
Comment (phunkyfish) [inline pvr.argustv/addon.xml.in]: Sorry, I spoke too soon. As you are adding functionality the version number should `19.2.0`. You would use `19.1.2` if you had fixed a bug.

File: pvr.argustv/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,11 @@
+v19.1.2
```
Comment (phunkyfish) [inline pvr.argustv/changelog.txt]: Sorry, I spoke too soon. As you are adding functionality the version number should `19.2.0`. You would use `19.1.2` if you had fixed a bug.

**Review by phunkyfish — CHANGES_REQUESTED** [review]

**Review by phunkyfish — APPROVED** [review]

**Pull request merged**